File: src/rootfs/etc/cyfs_gateway/add_chain_cmds/reject.py
import sys

if len(sys.argv) < 2:
    print("Usage: %s <domain>" % sys.argv[0])
    sys.exit(1)

dest = sys.argv[1]
print("call https-sni-probe && eq ${REQ.dest_host} \"%s\" && reject;" % dest)
sys.exit(0)

File: src/rootfs/etc/cyfs_gateway/add_chain_cmds/reject.js
export function main(argv) {
    if (argv.length < 1) {
        return 1;
    }

    let dest = argv[0];

    console.log(dest);
    let rule = `https-sni-probe && eq \$\{REQ.dest_host\} \"${dest}\" && reject;`;
    console.log(rule);
    return rule;
}
